kotlin/query-with-cql-filters/src/main/java/com/esri/arcgisruntime/sample/querywithcqlfilters/MainActivity.kt

```diff
@@ -55,7 +55,7 @@ class MainActivity : AppCompatActivity() {
         "F_CODE LIKE 'AQ%'",
         "{\"and\":[{\"eq\":[{ \"property\" : \"F_CODE\" }, \"AP010\"]},{ \"before\":" +
                 "[{ \"property\" : \"ZI001_SDV\"},\"2013-01-01\"]}]}",
-        ""
+        "Empty Query"
```

Comment:
perhaps "No query" is better than "Empty Query"?

Reply:
Added!